DEVELOPMENT.md

```diff
@@ -83,26 +103,50 @@ To set up locally, clone the repository and navigate to the `frontend` subdirect
     npm install
     ```
 
-4. Build the packages locally.
+4. Build the packages.
 
     ```bash
     npm run build
     ```
 
-    At this point the libraries should be ready to use in any of the projects in the `examples/` directory.
+    OR
 
-5. (Optional) Start the packages in watch mode.
+    Build in watch mode:
 
     ```bash
     npm run dev
     ```
 
-    Alternatively, you can publish the packages locally using Verdaccio:
+At this point the packages should be ready to use in the repository, such as linting/typechecking, IDE support, and use withinand of the projects in the [`./examples`](./examples) directory.
+
+5. (Optional) Link the `snapwp` command.
+
+    To test the local versions of [`./packages/cli`], you can link it to your global NPM packages:
+
+    ```bash
+    npm link snapwp
+    ```
+
+    For more information see [@todo]
```

Comment:
Update once bottom refs are reviewed.

Reply:
Link added to CLI's readme to direct the reader to usage docs for CLI.